File: packages/insomnia-httpsnippet/test/fixtures/output/java/unirest/application-json.java
HttpResponse<String> response = Unirest.post("http://mockbin.com/har")
  .header("content-type", "application/json")
  .body("{\"number\":1,\"string\":\"f\\\"oo\",\"arr\":[1,2,3],\"nested\":{\"a\":\"b\"},\"arr_mix\":[1,\"a\",{\"arr_mix_nested\":{}}],\"boolean\":false}")
  .asString();

File: packages/insomnia-httpsnippet/test/fixtures/output/node/unirest/application-json.js
var unirest = require('unirest');

var req = unirest('POST', 'http://mockbin.com/har');

req.headers({
  'content-type': 'application/json'
});

req.type('json');
req.send({
  number: 1,
  string: 'f"oo',
  arr: [1, 2, 3],
  nested: {
    a: 'b'
  },
  arr_mix: [
    1,
    'a',
    {
      arr_mix_nested: {}
    }
  ],
  boolean: false
});

req.end(function(res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
